collect_app/src/main/res/values/strings.xml

```diff
@@ -342,11 +342,12 @@
 <string name="map_sdk_selector_title">Mapping SDK</string>
 <string name="map_sdk_behavior">Mapping SDK</string>
 <string name="google_maps_sdk">Google Maps SDK</string>
-<string name="openstreetmap_sdk">OpenStreet Map SDK</string>
+<string name="openstreetmap_sdk">Other Map SDKs</string>
```

Comment:
This should still be OpenStreet Map SDK. The basemap options have changed but not the SDK.